pkg/kritis/review/review_test.go

```diff
@@ -370,3 +374,74 @@ func makeAtt(ids []string) []metadata.PGPAttestation {
 	}
 	return l
 }
+
+func TestGetAttestationAuthoritiesForISP(t *testing.T) {
+	authsMap := map[string]v1beta1.AttestationAuthority{
+		"a1": {
+			ObjectMeta: metav1.ObjectMeta{Name: "a1"},
+			Spec: v1beta1.AttestationAuthoritySpec{
+				NoteReference:        "provider/test",
+				PrivateKeySecretName: "test",
+				PublicKeyData:        "testdata",
+			}},
+		"a2": {
+			ObjectMeta: metav1.ObjectMeta{Name: "a2"},
+			Spec: v1beta1.AttestationAuthoritySpec{
+				NoteReference:        "provider/test",
+				PrivateKeySecretName: "test",
+				PublicKeyData:        "testdata",
+			}},
+	}
+	authMock := func(ns string, name string) (*v1beta1.AttestationAuthority, error) {
+		a, ok := authsMap[name]
+		if !ok {
+			return &v1beta1.AttestationAuthority{}, fmt.Errorf("could not find key %s", name)
+		}
+		return &a, nil
+	}
+
+	r := New(nil, &Config{
+		Auths: authMock,
+	})
+	tcs := []struct {
+		name        string
+		aList       []string
+		shdErr      bool
+		expectedLen int
+	}{
+		{
+			name:        "correct authorities list",
+			aList:       []string{"a1", "a2"},
+			shdErr:      false,
+			expectedLen: 2,
+		},
+		{
+			name:   "one incorrect authority in the list",
+			aList:  []string{"a1", "err"},
+			shdErr: true,
+		},
+		{
+			name:        "empty list should return nothing",
+			aList:       []string{},
+			shdErr:      false,
+			expectedLen: 0,
+		},
+	}
+
+	for _, tc := range tcs {
+		t.Run(tc.name, func(t *testing.T) {
+			isp := v1beta1.ImageSecurityPolicy{
+				Spec: v1beta1.ImageSecurityPolicySpec{
+					AttestationAuthorityNames: tc.aList,
+				},
+			}
+			auths, err := r.getAttestationAuthoritiesForISP(isp)
+			if (err != nil) != tc.shdErr {
+				t.Fatalf("expected review to return error %t, actual error %s", tc.shdErr, err)
+			}
+			if len(auths) != tc.expectedLen {
+				t.Fatalf("expected review to return error %t, actual error %s", tc.shdErr, err)
```

Comment:
Make this message more accurate. Also, use t.Errorf when possible in preference to t.Fatalf. They both behave the same in this example, however.